notesapi/v1/views.py

```diff
@@ -77,26 +82,17 @@ def get_from_es(self, *args, **kwargs):  # pylint: disable=unused-argument
             }
             query = query.highlight(**opts)
 
-        results = []
-        for item in query:
-            note_dict = item.get_stored_fields()
-            note_dict['ranges'] = json.loads(item.ranges)
-            # If ./manage.py rebuild_index has not been run after tags were added, item.tags will be None.
-            note_dict['tags'] = json.loads(item.tags) if item.tags else []
-            note_dict['id'] = str(item.pk)
-            if item.highlighted:
-                note_dict['text'] = item.highlighted[0].decode('unicode_escape')
-            if item.highlighted_tags:
-                note_dict['tags'] = json.loads(item.highlighted_tags[0])
-            results.append(note_dict)
-
-        return results
+        page = self.paginate_queryset(query)
+        serializer = NotesElasticSearchSerializer(page, many=True)
+        response = self.get_paginated_response(serializer.data)
+        return response
 
 
-class AnnotationListView(APIView):
+class AnnotationListView(GenericAPIView):
     """
     List all annotations or create.
```

Comment:
It would be nice if we can add proper doc strings for these views, which we will be using for documentation. Here is an example https://github.com/edx/edx-platform/pull/8082/files#diff-f0ef24d2da6e0cc7d7c5ce4d53fdda70R71